### PR TITLE
ONe compute plugin refactoring

### PR DESCRIPTION
**Pull request by pauloewerton**
Adding minor refactoring to the ONe compute plugin in advance of the unit tests refactoring.

### CODE REVIEW

File: src/main/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePlugin.java [hunk 1]
```diff
@@ -74,45 +77,111 @@ public boolean hasFailed(String cloudState) {
...
+		VirtualMachine virtualMachine = OpenNebulaClientUtil.getVirtualMachine(client, computeOrder.getInstanceId());
+		return this.doGetComputeInstance(virtualMachine);
+	}
+
+	@Override
+	public void deleteInstance(ComputeOrder computeOrder, CloudUser cloudUser) throws FogbowException {
+		Client client = OpenNebulaClientUtil.createClient(this.endpoint, cloudUser.getToken());
+		VirtualMachine virtualMachine = OpenNebulaClientUtil.getVirtualMachine(client, computeOrder.getInstanceId());
+		OneResponse response = virtualMachine.terminate(SHUTS_DOWN_HARD);
+		if (response.isError()) {
```
Comment (raonismaneoto) [inline src/main/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePlugin.java]: shouldn't an exception be thrown here?

File: src/main/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePlugin.java [hunk 2]
```diff
@@ -30,17 +30,20 @@
...
 
 	private static final int MB_CONVERT = 1024;
+	protected static final int DEFAULT_NUMBER_OF_INSTANCES = 1;
 
-	private static final String DEFAULT_ARCHITECTURE = "x86_64";
-	private static final String DEFAULT_GRAPHIC_ADDRESS = "0.0.0.0";
-	private static final String DEFAULT_GRAPHIC_TYPE = "vnc";
-	private static final String NETWORK_CONFIRMATION_CONTEXT = "YES";
-	private static final String NIC_IP_EXPRESSION = "//NIC/IP";
+	protected static final String DEFAULT_ARCHITECTURE = "x86_64";
```
Comment (raonismaneoto) [inline src/main/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePlugin.java]: I imagine you've changed the visibility of these variables to be able to use them in tests. If that's the case, maybe we should consider to have them in testUtils class, once they can be used in another plugin's tests, does it makes sense?

Comment (pauloewerton) [reply src/main/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePlugin.java]: yes, that's for testing. all of these constants are compute related, so i don't see other plugin tests using them. but i can sure move them if you really see fit. ;-)

File: src/main/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePlugin.java (hunk 1 above)
Comment (pauloewerton) [reply src/main/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePlugin.java]: sure

**Comment by pauloewerton** [comment]
@raonismaneoto thanks for your review. comments addressed.

**Pull request merged**